en/components/grids_templates/state_persistence.md

```diff
@@ -77,15 +124,82 @@ The usefullnes of thеse simple to use single-point API's allows you to achieve
 
   public saveGridState() {
       const state = this.state.getState() as string;
-      window.localStorage.setItem("grid1-state", state);
+      window.sessionStorage.setItem("grid1-state", state);
   }
 
   public restoreGridState() {
-      const state = window.localStorage.getItem("grid1-state");
+      const state = window.sessionStorage.getItem("grid1-state");
       this.state.setState(state);
   }
 ```
-. Depending on the scenario, the state can be saved to the browser `localStorage` or `sessionStorage` object, or saved in a database, cloud, passed on to a service, etc.
+
+#### Restoring columns
+
+[`IgxGridState`]({environment:angularApiUrl}/classes/igxgridstatedirective.html) will not persist columns templates, column formatters, etc. by default. Restoring any of therse can be achieved with code on application level. Let's show how to do this for templated columns:
+
```

Comment:
Here maybe it is a good idea to refer the limitation section.. maybe something like: 
Due to the limitation described below, [`IgxGridState`]({environment:angularApiUrl}/classes/igxgridstatedirective.html) will not persist columns templates, column formatters, etc. by default.  Restoring the state of any of these column features/properties can be achieved on application level.